mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1427,6 +1474,15 @@ func resourceSqlDatabaseInstanceUpdate(d *schema.ResourceData, meta interface{})
 	return resourceSqlDatabaseInstanceRead(d, meta)
 }
 
+func maintenanceVersionDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
+	if old > new {
```

Comment:
As discussed offline, this may not suppress the case, `"MYSQL_8_10_26.R20220508.01_09" => "MYSQL_8_7_37.R20210508.01_03"`.

Reply:
Fixed. PTAL!